akka-actor/src/main/scala/akka/actor/ActorCell.scala

```diff
@@ -686,7 +686,9 @@ private[akka] class ActorCell(
     case _                               =>
   }
 
-  final protected def clearActorFields(): Unit = {
+  @InternalStableApi
+  @silent("never used")
+  final protected def clearActorFields(actorInstance: Actor, recreate: Boolean): Unit = {
```

Comment:
any mima filter that should be removed now?

Reply:
Good point. Removed the line which is now obsolete.